admiral/pkg/clusters/envoyfilter.go

```diff
@@ -0,0 +1,165 @@
+package clusters
+
+import (
+	"fmt"
+	"github.com/gogo/protobuf/types"
+	v1 "github.com/istio-ecosystem/admiral/admiral/pkg/apis/admiral/v1"
+	"github.com/istio-ecosystem/admiral/admiral/pkg/controller/admiral"
+	"github.com/istio-ecosystem/admiral/admiral/pkg/controller/common"
+	log "github.com/sirupsen/logrus"
+	"istio.io/api/networking/v1alpha3"
+	networking "istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+)
+
+var (
+	getSha1 = common.GetSha1
+)
+const wasmPath = "/etc/istio/extensions/dynamicrouter.wasm"
```

Comment:
Can this be moved to the RoutingPolicy as well?

Reply:
moved, but it will be there in user config so we need to make them aware